### DISCUSSION TITLE
SEO text

### DISCUSSION THREAD

**Discussion by Mojitoooooooooooooooo** [GetPublii/Publii#1477] [Help]
Help me, please.
How to place seo text on the main page? I can do it in html, but the text is overwritten during synchronization.

**Comment by internettips** [comment]
More details needed.

'SEO text' can cover many different areas -- though the Publii paid themes seem to cover much of what is probably needed, perhaps many of the free themes too. So much depends on which SEO text you want to add. 

Also, which theme are you using?

Do you have a live URL you can share?

**Reply by Mojitoooooooooooooooo** [reply]
I am using QF Fashion theme, in this theme there is no option to add text on the main page. I was planning to add text at the bottom of the site.

**Reply by candidexmedia** [reply]
@Mojitoooooooooooooooo So, you are looking to add just regular text on your home page and not SEO text hidden in markup (ie meta descriptions), correct? Or are you referring to the footer?

If you have a visual of what you're trying to achieve, that would also help.

**Comment by internettips** [comment]
@Mojitoooooooooooooooo 

Some more background information that may help.

In the Publii app, go to:

Theme > Theme settings > Custom settings > Layout > Front page, for the main make-up of your home page, the software shows that you can either use a mix of samples of existing posts ('Latest posts') -- based on tag names that apply to each post -- or you can choose a single page / post that you've already created and have Publii use this as your home page.

For both of these options, as you're creating your pages, if you fill in the various boxes for SEO stuff, Publii automatically applies the relevant SEO elements to your home page. So much of the SEO work can be done for you providing you create your posts fully.

If you wanted a somewhat different layout the the basic design of the default QF theme, as shown in the demo at: https://demo.getpublii.eu/themes/qf/ , you can modify the design but this does involve some more work. Plus, if you want to make sure any of such latest changes stay in place when theme updates and Publii updates become available, you'll need to do additional steps.

**Comment by Mojitoooooooooooooooo** [comment]
Thank you all for your help!

**Comment by Mojitoooooooooooooooo** [comment]
Thank you all for your help!
